en/flight_modes/land.md

```diff
@@ -0,0 +1,43 @@
+# LAND Flight Mode
+
+The LAND flight mode causes the vehicle to land at the position where the mode was engaged.
+
+The specific behaviour for each vehicle type is described below.
+
+> **Note** 
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+
+## Multi-Copter (MC)
+
+The vehicle will land at the location at which the mode was engaged. The vehicle descends at the rate specified in `MPC_LAND_SPEED` until it hits the ground.
+
+Landing is affected by the following parameters:
+
+Parameter | Description
+--- | ---
+[MPC_LAND_SPEED](../advanced_config/parameter_reference.md#MPC_LAND_SPEED) | The rate of descent during landing. This should be kept fairly low as the ground conditions are not known.
+
+## Fixed Wing (FW)
+
+The vehicle turns and lands at the location at which the mode was engaged. 
+
+> **Note** The vehicle doesn't necessarily know where the ground is, and will therefore follow a fixed trajectory for the whole landing approach (no flare).
+
+Landing is affected by the following parameters:

```

Comment:
@dagar FW LAND mode simply follows a trajectory to ground. Would it be correct to say therefore that most of the parameters here (e.g. FW_LND_HVIRT, FW_LND_FLALT, etc are not relevant). Can you confirm which ones are/are not relevant and I'll delete them from this list.